5.5/test/run

```diff
@@ -92,6 +104,17 @@ run_tests() {
 	create_container $name $envs
 	CONTAINER_IP=$(get_container_ip $name)
 	test_connection $name
+	echo "  Testing login accesses"
+	assert_login_access $USER $PASS true
+	assert_login_access $USER "${PASS}_foo" false
+	if [ -v ROOT_PASS ]; then
+		assert_login_access root $ROOT_PASS true
+		assert_login_access root "${ROOT_PASS}_foo" false
+	else
+		assert_login_access root "foo" false
+		assert_login_access root "" false
```

Comment:
Those are not local connections, the mysql commands are executed from the host. By docker, using the same image, but from the host.